.github/workflows/test.yaml

```diff
@@ -51,6 +51,9 @@ jobs:
         run: make test
   golden:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        test: [defaults, policies]
```

Comment:
Please use `commodore component update -t policies --no-commit /path/to/component` to ensure the new test case is added everywhere. (note: `--no-commit` is currently required because of https://github.com/projectsyn/commodore/issues/626).